.github/workflows/test_pytest.yaml

```diff
@@ -24,9 +24,12 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install -r requirements-tests.txt
-      - name: Build RocketPy
+      - name: Build RocketPy (without optional dependencies)
         run: |
-          pip install -e.[all]
+          pip install -e.
+      - name: Import rocketpy in python and test if it works
+        run: |
+          python -c "import sys, rocketpy; print(f'{rocketpy.__name__} running on Python {sys.version}')"
       - name: Test with pytest
         run: |
           pytest
```

Comment:
Nonetheless, I will review this PR. There are some issues with the current way the workflow install is designed:

1. `pip install -e .` I don't think it makes much sense to build a development/editable (`-e` flag) install since there will be no edits to the source code on the workflow. The tests should be performed in a regular install, since the editable one is intended for development and has some issues, such as, preventing a clean uninstall;
    - For references, compare [regular install](https://pip.pypa.io/en/stable/topics/local-project-installs/#regular-installs) and [editable install](https://pip.pypa.io/en/stable/topics/local-project-installs/#editable-installs).
2. The `requirements-test.txt` install should be performed after the `rocketpy` import test, so that we can make sure that no requirements that are test only are filling a gap in the install requirements.

Overall, good work on testing the project `import`, this will help to catch some bugs that have been overlooked.

Reply:
1. You're 100% right.
2. Amazingly pointed out, thanks a lot for noticing.

I just fixed it in the last commit.